README.md

```diff
@@ -20,8 +20,11 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 
 ## Release Notes
 
+2.0.11:
+  - pac CLI 1.28.2 (September refresh, see release notes on [nuget.org](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/))
```

Comment:
For vscode extension release here are the release notes:

- In Web, now Supporting image edit capability using Vscode extensions for Image editing (supported file extensions are png, jpg, webp, bmp, tga, ico, dib, jif, jpe, tpic, jpe, jpeg)
- IN Web, File explorer now supports Power Pages actions for "Going back to Power pages Studio" and "Preview" of site's runtime instance.